pytero/app.py

```diff
@@ -632,17 +632,26 @@ async def create_node_allocation(
         ip: str,
         ports: list[str],
         alias: str = None
-    ) -> Allocation:
-        data = await self._http.post(
+    ) -> None:
+        """ Create a new allocated node.
```

Comment:
It's creating a node allocation, not a node.
```suggestion
        """ Create a new node allocation.
```